.github/workflows/workflow-vercel-deployment.yml

```diff
@@ -29,15 +26,31 @@ jobs:
   deploy:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+
+      - name: checkout code
+        uses: actions/checkout@v3
+
+      - name: install node
+        uses: actions/setup-node@v3
+        with:
+          cache: npm
+          node-version-file: "package.json"
+
+      - name: install npm
+        run: npm install -g npm@9.8.1
```

Comment:
if your error is this as reported in the issue:

```
npm ERR! notsup Required: {"node":">=18.0.0 <1[9](https://github.com/ai-cfia/finesse-frontend/actions/runs/7142466425/job/19451818974#step:5:10).0.0","npm":">=10.0.0 <11.0.0"}
```

How does 9.8.1 fulfill this requirement? it's below 10?

Reply:
copied the wrong error my bad, this one was from me changing things for debug
updated to issue to this
npm ERR! notsup Required: {"npm":"^9.8.1","node":"^18.[16](https://github.com/ai-cfia/finesse-frontend/actions/runs/7142585744/job/19452590732#step:6:17).0"}
npm ERR! notsup Actual:   {"npm":"10.2.3","node":"v18.19.0"}